frontend/src/components/pages/MockSearch/MockSearchPage.tsx

```diff
@@ -0,0 +1,204 @@
+import { Box } from "@chakra-ui/react";
+import React from "react";
+
+import { Author, Book } from "../../../types/BookTypes";
+import { Review } from "../../../types/ReviewTypes";
+import ReviewThumbnail from "./ReviewThumbnail";
+
+const mockGenericBookData: Book = {
```

Comment:
Could you maybe move this mock data into a separate file? There should be a `mock` folder somewhere

Reply:
Couldn't find the mock folder but created a file called mockReviews.ts and put all of the mock books and reviews in there!